lib/cldr-plurals/javascript_emitter.rb

```diff
@@ -84,6 +82,11 @@ def emit_value_set(value_set, operand, operator)
         end
       end
 
+      def emit_range_check(range, operand)
+        # a..b represents all *integers* between a and b, inclusive.
+        (range.start..range.finish).to_a.map { |v| "(#{emit(operand)} == #{v})" }.join(" || ")
```

Comment:
Curious what you meant by "this still gets ugly in the 11.99 cases."

I'm fine with leaving this as `==` since I don't think it really matters (all operands should be the same type, i.e. doubles). Just a small thing more to follow convention than function.